plugin/web/extensions/sidebar/types.ts

```diff
@@ -114,6 +114,14 @@ export type ReearthApi = {
     start?: string;
     stop?: string;
   };
+  light?: {
+    type?: "sunLight" | "directionalLight";
+    direction_x?: number;
+    direction_y?: number;
+    direction_z?: number;
+    color?: string;
+    intensity?: number;
```

Comment:
```suggestion
    lightType?: "sunLight" | "directionalLight";
    lightDirectionX?: number;
    lightDirectionY?: number;
    lightDirectionZ?: number;
    lightColor?: string;
    lightIntensity?: number;
```

Sorry, I forgot to commnent. Can you rename the properties?
https://github.com/reearth/reearth-web/commit/8b3a189a6ae4deeb9a9fe7e680324eed1bf7aa24

Reply:
Yes i notice this on test. Already updated.